samples/DispatcherSample/DispatcherSample.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
```

Comment:
This should have project references to the other projects you added

Reply:
And get rid of the metapackage here, we need to keep this lean, and it needs to build with latest bits.